q2_feature_classifier/_taxonomic_classifier.py

```diff
@@ -69,7 +69,17 @@ def _1(dirfmt: TaxonomicClassiferTemporaryPickleDirFmt) -> Pipeline:
         tmpdir = model.DirectoryFormat()
         dirname = str(tmpdir)
         tar.extractall(dirname)
-        pipeline = joblib.load(os.path.join(dirname, 'sklearn_pipeline.pkl'))
+        try:
+            pipeline = joblib.load(os.path.join(dirname,
+                                                'sklearn_pipeline.pkl'))
+        except MemoryError:
```

Comment:
Let's add a test for this, by [mocking](https://docs.python.org/3.6/library/unittest.mock.html) joblib.load to raise this memory error. Please let one of us know if you need a hand with this!

Reply:
I was able to work on this with @Oddant1. If you like the changes, I would like to add him as a co-author.

Reply:
Is this where the action generally fails at? Looking around on the forum, it seems like more often than not, the loading succeeds, but the action fails later when running the meat of the command. Its possible your VBox testing is triggering this failure point, which is a good catch, but, I suspect it isn't the typical failure point.

Reply:
I searched for the error on the forum, and one of the first hits was: https://forum.qiime2.org/t/memory-error-qiime-feature-classifier-classify-sklearn/9066

If you look at the error traceback, the first "qiime 2" code in the call stack is:

```bash
File "/home/patricia/miniconda2/envs/qiime2-2019.1/lib/python3.6/site-packages/q2_feature_classifier/
skl.py ", line 45, in predict
```

That means that this is the "last" bit of the plugin's code to be called before the error crops up. This seems like a good place to start.